pandas/tests/test_generic.py

```diff
@@ -1859,6 +1859,23 @@ def test_pipe_panel(self):
         with tm.assertRaises(ValueError):
             result = wp.pipe((f, 'y'), x=1, y=1)
 
+    # GH 11150
+    def test_pct_change_frame(self):
+        pnl = DataFrame([np.arange(0, 40, 10), np.arange(0, 40, 10), np.arange(0, 40, 10)]).astype(np.float64)
+        pnl.iat[1,0] = np.nan
+        pnl.iat[1,1] = np.nan
```

Comment:
I mean that this function needs to move to ``tests\test_frame.py`` (and not in ``test_generic.py``)